README.md

```diff
@@ -80,5 +86,6 @@ sampleEachSecond([of(1), of(2)])
 // 1, 2
 // Seconds Passed: 2
 // 2, 4
-
+ 
 ```
+ Its functionality is currently available in [@most/core](https://mostcore.readthedocs.io/en/latest/api.html#sample), and will also be available in most 2.0, when it's released. 
```

Comment:
An extra copy of the deprecation ... for good measure 😉

🔪

Reply:
i dont even know how it snuck in, i blame it on lack of sleep.. so Miles is the culprit 😛